sites/public/__tests__/pages/applications/preferences/all.test.tsx

```diff
@@ -2,6 +2,14 @@ import React from "react"
 import { setupServer } from "msw/lib/node"
 import { mockNextRouter, render } from "../../../testUtils"
 import ApplicationPreferencesAll from "../../../../src/pages/applications/preferences/all"
+import { AppSubmissionContext } from "../../../../src/lib/applications/AppSubmissionContext"
```

Comment:
nit: imports aren't alphabetized

Reply:
Is that a new standard we are following? I typically don't alphabetize the imports, but I can start doing that if we want to.

Reply:
Idk if we ever formalized it but I know Emily and I have been doing that lately. This is fine to go out and we can discuss later

Reply:
Sounds good!